Topics/Development_Process/Standards_For_Best_Practices.md

```diff
@@ -0,0 +1,59 @@
+# Utilizing Standards to Adopt Best-Practices in Software Engineering
+
+## Table of Contents
+
+### [What Is a Standard?](#what-is-a-standard-1)
+
+### [Why Are Standards Important?](#why-are-standards-important-1)
+
+### [How Are Standards Created?](#how-are-standards-created-1)
+
+### [Why is This Relevant to CSC301 Students?](#why-is-this-relevant-to-csc301-students-1)
+
+### [Examples of Standards of Relevance](#examples-of-standards-of-relevance-1)
+
+### [Other Useful Standards](#other-useful-standards-1)
+
+### [Sources](#sources-1)
+
+### What Is a Standard?
+
+According to the Standards Council of Canada (SCC), a standard is defined "as a document, established by consensus and approved by a recognized body, that provides for common and repeated use, rules, guidelines or characteristics for activities or their results, aimed at the achievement of the optimum degree or order in a given context."
```

Comment:
remove the 'as' from the parenthesis (e.g. ... is defined as "a document, established...")